KvNGCzA/Hello-Books-BE
# #166313092 create user model

What does this PR do?
Create a user model

Description of Task to be completed?
- Create a user model
- Add unit test for model

How should this be manually tested?
- Use **git pull** to pull branch to local branch
- Use **npm install** to download all necessary packages
- Update default config file to use environment variables of the project
- Ensure your database is created or you can use **sequelize db:create** if database does not exist
- Use **npm run db:roll** to create table in database

Any background context you want to provide?
NA

What are the relevant pivotal tracker stories?
[#166313092](https://www.pivotaltracker.com/story/show/166313092)

Screenshots (if appropriate)
NA

Questions:
NA
